### PR TITLE
Feat/steady state analysis

### PR DESCRIPTION
**Pull request by gurdeep330**
# For authors

## Description
Please:
1. Provide a summary of the modifications made and any associated issue (if applicable).
2. Include relevant context and motivation for the changes.
3. If this relates to a change in any website's frontend, kindly attach a screenshot of the adjustment from your localhost.
4. List any dependencies necessary for implementing this change.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests you conducted to verify your changes. These may involve creating new test scripts or updating existing ones.
- [ ] Added new test(s) in the ```tests``` folder
- [ ] Added new function(s) to an existing test(s) (e.g.: ```tests/testX.py```)
- [ ] No new tests added (Please explain the rationale in this case)

## Checklist
- [ ] My code follows the style guidelines mentioned in the Code/DevOps guides
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. MkDocs)
- [ ] My changes generate no new warnings
- [ ] I have added or updated tests (in the ```tests``` folder) that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# For reviewers

## Checklist pre-approval
- [ ] Is there enough documentation?  
- [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
- [ ] Does the test(s) successfully test edge/corner cases?
- [ ] Does the PR pass the tests? (if the repository has continuous integration)

## Checklist post-approval
- [ ] Does this PR merge ```develop``` into ```main```? If so, please make sure to add a prefix (feat/fix/chore) and/or a suffix BREAKING CHANGE (if it's a major release) to your commit message.
- [ ] Does this PR close an issue? If so, please make sure to descriptively close this issue when the PR is merged.

## Checklist post-merge
- [ ] When you approve of the PR, merge and close it (Read this [article](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github) to know about different merge methods on GitHub)
- [ ] Did this PR merge ```develop``` into ```main``` and is it suppose to run an automated release workflow (if applicable)? If so, please make sure to check under the "Actions" tab to see if the workflow has been initiated, and return later to verify that it has completed successfully.

### CODE REVIEW

File: aiagents4pharma/talk2biomodels/states/state_talk2biomodels.py [hunk 1]
```diff
@@ -12,13 +12,14 @@ class Talk2Biomodels(AgentState):
...
     # which is not supported by the StateGraph.
     # Therefore, we need to use Annotated to specify
     # the operator for the sbml_file_path field.
     # https://langchain-ai.github.io/langgraph/troubleshooting/errors/INVALID_CONCURRENT_GRAPH_UPDATE/
+    model_id: Annotated[list, operator.add]
     sbml_file_path: Annotated[list, operator.add]
     dic_simulated_data: Annotated[list[dict], operator.add]
-    llm_model: str
+    dic_scanned_data: Annotated[list[dict], operator.add]
+    dic_steady_state_data: Annotated[list[dict], operator.add]
```
Comment (dmccloskey) [inline aiagents4pharma/talk2biomodels/states/state_talk2biomodels.py:25]: Is there a reason to have `dic_simulated_data`, `dic_scanned_data`, and `dic_steady_state_data`, instead of a single `dic_simulated_data` struc?. From my understanding, these all have similar shape (i.e., time vs concentration) and could be differentiated by the experiment_id key?

Comment (dmccloskey) [reply aiagents4pharma/talk2biomodels/states/state_talk2biomodels.py:25]: Based on our conversation over Teams, this can be dealt with in this PR or a separate PR: up to you.

**Review by dmccloskey — APPROVED** [review]
Looks good to me 👍
